### PR TITLE
use/add on-chain values in getting-started doc via RPC

### PR DESCRIPTION
**Pull request by alfarok**
### Description

Replaces hard-coded values w/ on-chain equivalents.

### Follow-up

Fetch the minimum active nomination from on-chain when available via Polkadot-JS.  See #4123 for additional details.

### CODE REVIEW

File: docs/general/getting-started.md [hunk 1]
```diff
@@ -191,15 +194,24 @@ functionalities on Polkadot.
   have lesser DOT than the minimum active nomination, please consider contributing to
   [nomination pools](../learn/learn-nomination-pools.md). Learn more about
   [becoming a nominator](../learn/learn-nominator.md).
-- **20 DOT**: you can register an [on-chain identity](../learn/learn-identity.md), vote for
-  [council](../maintain/maintain-guides-how-to-vote-councillor.md), or create a
-  [proxy account](../learn/learn-proxies.md).
 
-- **200 DOT**: you can create your own [nomination pool](../learn/learn-nomination-pools.md).
+- **Roughly 20 DOT**:
```
Comment (profradha) [inline docs/general/getting-started.md]:
```suggestion
- **Around 20 DOT**:
```

File: docs/general/kusama/kusama-getting-started.md [hunk 2]
```diff
@@ -181,6 +183,45 @@ nominating), governance, parachain auctions, basic transfers and everything else
...
 
+### What can I do with my KSM?
+
+KSM is the native token of the Kusama Network. KSM can be used for transaction fees, staking,
+governance, acquisition of a parachain slot and for enabling several key functionalities on Kusama.
+
+- {{ kusama: __<RPC network="kusama" path="consts.balances.existentialDeposit" defaultValue={333333333} filter="humanReadable"/>:__ :kusama }}
+  the minimum balance required to have an active account on Kusama Network. If your account balance
+  drops below the minimum, your account will be reaped. Learn more about
+  [Accounts](../learn/learn-accounts.md) and the
```
Comment (profradha) [inline docs/general/kusama/kusama-getting-started.md]: These markdown links would not work

**Review by DrW3RK — CHANGES_REQUESTED** [review]
Pretty cool! 

Although, adjust the markdown doc links for Kusama guide appropriately!

File: docs/general/kusama/kusama-getting-started.md (hunk 2 above)
Comment (alfarok) [reply docs/general/kusama/kusama-getting-started.md]: good catch! should be fixed now

**Review by DrW3RK — APPROVED** [review]

**Pull request merged**